changelog.d/otlp_decoding.enhancement.md

```diff
@@ -0,0 +1,8 @@
+The `opentelemetry` source now supports a new decoding mode which can be enabled by setting `use_otlp_decoding` to `true`. In this mode,
+all events will preserve the [OTLP](https://opentelemetry.io/docs/specs/otel/protocol/) format. These events can be forwarded directly to
+the `opentelemetry` sink without modifications.
+
+A caveat here is that OTLP metrics and Vector metric format differ and thus we treat as logs as they come out the source. These events
+cannot be used with existing metrics transforms. However, these can be ingested by the OTEL collectors as metrics.
```

Comment:
```suggestion
The OTLP metric format and the Vector metric format differ, so Vector treats [OTLP formatted events??] as logs as they come out the source. These events
cannot be used with existing metrics transforms. However, they can be ingested by the OTEL collectors as metrics.
```
I tried to clarify these lines. Please feel free to correct my wording, as I wasn't sure what entities are treated as logs.